### PR TITLE
#71 #72 Audio Settings. Music pack support. Boundary Map Lookup.

### PR DESCRIPTION
**Pull request by Hubcapp**

### CODE REVIEW

**Review by sean-niemann — COMMENTED** [review]
Looks good!

File: src/Client/ConfigWindow.java [hunk 1]
```diff
@@ -2863,6 +3304,84 @@ public void synchronizeGuiValues() {
...
+    // Audio tab
+    audioPanelEnableMusicCheckbox.setSelected(Settings.CUSTOM_MUSIC.get(Settings.currentProfile));
+    audioPanelLouderSoundEffectsCheckbox.setSelected(
+        Settings.LOUDER_SOUND_EFFECTS.get(Settings.currentProfile));
+    audioPanelOverrideAudioSettingCheckbox.setSelected(
+        Settings.OVERRIDE_AUDIO_SETTING.get(Settings.currentProfile));
+    audioPanelOverrideAudioSettingOnButton.setSelected(
+        Settings.OVERRIDE_AUDIO_SETTING_SETTING_ON.get(Settings.currentProfile));
+    audioPanelOverrideAudioSettingOffButton.setSelected(
+        !Settings.OVERRIDE_AUDIO_SETTING_SETTING_ON.get(Settings.currentProfile));
```
Comment (lmsv-mx123) [inline src/Client/ConfigWindow.java:3316]: seems like typo, or if correct 3313 and 3314 removed

Comment (lmsv-mx123) [reply src/Client/ConfigWindow.java:3316]: nvm ignore

**Review by lmsv-mx123 — COMMENTED** [review]
From code reading didnt find some issue

**Pull request merged**